mrmustard/math/tensorflow.py

```diff
@@ -378,7 +389,7 @@ def hermite_renormalized(
             The renormalized Hermite polynomial of given shape.
         """
         _A, _B, _C = self.asnumpy(A), self.asnumpy(B), self.asnumpy(C)
-        G = strategies.vanilla(tuple(shape), _A, _B, _C)
+        G = Main_julia.vanilla(tuple(shape), _A, _B, _C.item())
```

Comment:
Do we want every call to `hermite_renormalized` run in julia? I would think sometimes we still want the old numba code to run, where there's no need for the extra precision

Reply:
I'll add a keyword argument `precision`. And if precision==float64, then I redirect to the Numba code.